compose/use-secrets.md

```diff
@@ -74,9 +74,13 @@ volumes:
 In the advanced example above:
 
 - The `secrets` attribute under each service defines the secrets you want to inject into the specific container.
-- The top-level secrets section defines the variables `db_password` and `db_root_password` and provides the file that populates their values.
+- The top-level `secrets` section defines the variables `db_password` and `db_root_password` and provides the `file` that populates their values.
 - The deployment of each container means Docker creates a temporary filesystem mount under `/run/secrets/<secret_name>` with their specific values.
 
+> **Note**
+>
+> The [_FILE](https://hub.docker.com/_/mysql) convention is specific to mysql image.
```

Comment:
We should make it clear that this is specific to the `mysql` image used in this example, as e.g. https://hub.docker.com/_/postgres also supports this convention, as does much of DOI (cc @tianon).

Maybe something more like:
```suggestion
> The `_FILE` environment variables demonstrated here are a convention used by some images, including Docker Official Images like [mysql](https://hub.docker.com/_/mysql) and [postgres](https://hub.docker.com/_/postgres).
```

Reply:
Looks good to me.